### PR TITLE
Fix old TODO (STATS::rangemax_)

### PR DESCRIPTION
**Pull request by stweil**
Signed-off-by: Stefan Weil <sw@weilnetz.de>

### CODE REVIEW

**Comment by amitdo** [comment]
ClFuzz failed.

File: src/ccstruct/statistc.h [hunk 1]
```diff
@@ -30,23 +30,20 @@ namespace tesseract {
 class TESS_API STATS {
 public:
   // The histogram buckets are in the range
-  // [min_bucket_value, max_bucket_value_plus_1 - 1] i.e.
-  // [min_bucket_value, max_bucket_value].
+  // [min_bucket_value, max_bucket_value - 1].
```
Comment (stweil) [inline src/ccstruct/statistc.h:33]:
```suggestion
  // [min_bucket_value, max_bucket_value].
```

File: src/ccstruct/statistc.h [hunk 2]
```diff
@@ -30,23 +30,20 @@ namespace tesseract {
...
   // Any data under min_bucket value is silently mapped to min_bucket_value,
-  // and likewise, any data over max_bucket_value is silently mapped to
-  // max_bucket_value.
+  // and likewise, any data over max_bucket_value - 1 is silently mapped to
+  // max_bucket_value - 1.
```
Comment (stweil) [inline src/ccstruct/statistc.h:36]:
```suggestion
  // and likewise, any data over max_bucket_value is silently mapped to
  // max_bucket_value.
```

**Comment by amitdo** [comment]
Stefan, reviewing your own PR? :laughing: 

Is this ready now?

**Comment by stweil** [comment]
The change is rather large, and there is always the risk that some line was missed or changed incorrectly. Therefore reviewing is essential. As you had noticed, the first version failed during the continuous integration, so I had to review my own pull request. Now I think it is ready, but suggest to delay it after 5.0.1 to minimize any risk.

**Pull request merged**